### DISCUSSION TITLE
使用 CDN 加速 GitHub Pages 博客的静态资源 — 码志

### DISCUSSION THREAD

**Discussion by mzlogin** [mzlogin/blog-comments#234] [Announcements]
https://mazhuang.org/2020/05/01/cdn-for-github-pages/ 

 挺久以前就有网友给我的 GitHub Pages 博客模板提 Issue，说希望能增加 CDN 用于加速静态资源的加载，由于懒，一直没有动。

**Comment by NanyuKe** [comment]
不错😀😀😀

**Comment by mowenliunian** [comment]
感谢分享
